paparazzi/src/main/java/app/cash/paparazzi/HtmlReportWriter.kt

```diff
@@ -92,7 +92,7 @@ public class HtmlReportWriter @JvmOverloads constructor(
       val snapshotDir = if (fps == -1) imagesDirectory else videosDirectory
       val goldenDir = if (fps == -1) goldenImagesDirectory else goldenVideosDirectory
       val hashes = mutableListOf<String>()
-      val snapshotTmpFile = File(snapshotDir, snapshot.toFileName(extension = "temp.png"))
```

Comment:
needed a better delimeter to denote method names from custom names appended at the end

Reply:
what's the difference with/without this delimiter change?

Reply:
before: `delta-app.cash.paparazzi.sample_LaunchViewTest_pixel3_differentThemes_light.png`

after: `delta-app.cash.paparazzi.sample_LaunchViewTest_pixel3_differentThemes-light.png`

Reply:
let's keep this as is, although I do think output filename formatting is definitely an improvement to make down the road.

Reply:
done, reverted the change. Turned out I didn't need this change anyway since snapshots are aggregated by test class and method name.